package/tests/test_integrations/test_hooks.py

```diff
@@ -0,0 +1,24 @@
+from unittest import mock
+
+from kedro_viz.integrations.kedro.hooks import dataset_stats_hook
+
+
+def test_after_dataset_loaded(example_data_frame, dataset_name: str = "raw_data"):
+    # Test for pd.dataframe
+    assert (
+        dataset_stats_hook.after_dataset_loaded(dataset_name, example_data_frame)
+        is None
+    )
+    # Test for Transcoded data node
+    transcoded_dataset_name = "raw_data@pandas1"
+    assert (
+        dataset_stats_hook.after_dataset_loaded(
+            transcoded_dataset_name, example_data_frame
+        )
+        is None
+    )
+
```

Comment:
What are you testing here?

Reply:
This was primarily to get the code coverage and it covers the utility function where we strip transcoded name. As you said the test cases are really not adding any value here. I need to rewrite them.